### ISSUE TITLE
layer-wise learning rate

### ISSUE THREAD

**Issue by coldgemini** [fizyr/keras-retinanet#229]
is there a way to control the learning rate per layer when doing transfer learning?
thanks.

**Comment by smehida** [comment]
you can look at this [#3004](https://github.com/keras-team/keras/pull/3004).
But as an alternative solution you can use Adam and Adadelta optimizer and tune their parameters as you want. I think this can be a better solution.

**Comment by hgaiser** [comment]
This isn't a `keras-retinanet` issue, but a Keras / Tensorflow issue. I'll close this for now, my advise is to see if one of those issue boards can help you further.

**Comment by halidziya** [comment]
https://github.com/keras-team/keras/blob/b88bbbab2695dd0e24905417006d2240c6595481/keras/optimizers.py#L195

Maybe something like this if the lr is given as array: 

        for i, p, g, m in zip(range(len(param)),params, grads, moments):
            v = self.momentum * m - lr[i] * g  # velocity
            self.updates.append(K.update(m, v))
